qutip/tests/test_steadystate.py

```diff
@@ -487,7 +487,7 @@ def test_driven_cavity_power_gmres():
     c_ops = [np.sqrt(Gamma) * a]
     M = build_preconditioner(H, c_ops, method='power')
     rho_ss = steadystate(H, c_ops, method='power-gmres', M=M, mtol=1e-1,
-                         use_precond=1)
+                         use_precond=1, maxiter=10000)
```

Comment:
Why did you make this change?

Reply:
This test cannot get passed on my machine because ``maxiter`` is too small.

Reply:
It passes on every other test on Travis without this and we set the default for general usage to `maxiter=1000`, so we need to work out why it is different on your machine.